docs/vars.md

```diff
@@ -205,6 +205,10 @@ Stack](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/dns-stack.m
 
 * *kubelet_streaming_connection_idle_timeout* - Set the maximum time a streaming connection can be idle before the connection is automatically closed.
 
+* *kubelet_image_gc_high_threshold* - Set the percent of disk usage before which image garbage collection is never run.
+
+* *kubelet_image_gc_low_threshold* - Set the percent of disk usage before which image garbage collection is never run .
```

Comment:
Could we describe different explanation with the above `kubelet_image_gc_high_threshold`?
Current description is completely same and it is hard to know the difference from that.
In addition, it is nice to describe some example values for those options.
(I was wondering we need to specify `90` or `90%`)

Reply:
Sorry, it's a typo. 

It is fixed, and the commits are squashed.